interpreter/exec/eval.ml

```diff
@@ -282,27 +282,27 @@ let rec step (c : config) : config =
         with exn -> vs', [Trapping (numeric_error e.at exn) @@ e.at])
 
       | Ternary ternop, v3 :: v2 :: v1 :: vs' ->
-        (try Eval_numeric.eval_ternop ternop v1 v2 v3 :: vs', []
+        (try Eval_simd.eval_ternop ternop v1 v2 v3 :: vs', []
```

Comment:
Since this actually is a SIMD-specific, maybe rename to SimdTernary? If just named Ternary, it should be handled analogously to Binary.

Reply:
Yea good point, renamed to SimdTernary (otherwise we will need to create new ternop for all the other numeric types, which contain nothing...). Thanks!